controllers/common.go

```diff
@@ -92,3 +102,35 @@ func reconcileDesiredObjects(ctx context.Context, kubeClient client.Client, logg
 	}
 	return nil
 }
+
+func enabledAcceleratedComputeByAgentConfig(ctx context.Context, c client.Client, log logr.Logger) bool {
+	agentResource := getAmazonCloudWatchAgentResource(ctx, c)
+	// missing feature flag means it's on by default
+	featureConfigExists := strings.Contains(agentResource.Spec.Config, acceleratedComputeMetrics)
```

Comment:
Is there a reason why we aren't checking `conf.Logs.LogMetricsCollected.Kubernetes.AcceleratedComputeMetrics` to determine if the feature config exists?

Reply:
it's a boolean flag defaulting to false if config doesn't exist where the expected behavior is "true" when missing the config. This is just to distinguish between missing config vs. setting it to "false" to turn the feature off.